.vscode/settings.json

```diff
@@ -1,13 +1,10 @@
 {
   // adds tailwind autocomplete in other places
+  "tailwindCSS.classAttributes": ["class", "className", "class:list"],
   "tailwindCSS.experimental.classRegex": [
     ["(?:classnames|classNames|cx|clsx)\\(([^)]*)\\)", "(?:')((?:\\'|[^'])*)(?:')"],
     ["(?:classnames|classNames|cx|clsx)\\(([^)]*)\\)", "(?:\")((?:\\\"|[^\"])*)(?:\")"],
-    ["(?:classnames|classNames|cx|clsx)\\(([^)]*)\\)", "(?:`)((?:\\`|[^`])*)(?:`)"],
-    // match strings inside of astro's `class:list={[...]}` syntax
-    ["(?:class:list\\s*=\\s*\\{\\[)([^\\]]*)(?:\\]\\})", "(?:')((?:\\'|[^'])*)(?:')"],
-    ["(?:class:list\\s*=\\s*\\{\\[)([^\\]]*)(?:\\]\\})", "(?:\")((?:\\\"|[^\"])*)(?:\")"],
-    ["(?:class:list\\s*=\\s*\\{\\[)([^\\]]*)(?:\\]\\})", "(?:`)((?:\\`|[^`])*)(?:`)"]
+    ["(?:classnames|classNames|cx|clsx)\\(([^)]*)\\)", "(?:`)((?:\\`|[^`])*)(?:`)"]
```

Comment:
I remembered classAttributes exists and figured regexes might be slowing the editor down